helm-go-lib/README.md

```diff
@@ -1 +1,2 @@
-Run `go get helm.sh/helm/v3@v3.2.0` to get dependencies
\ No newline at end of file
+Run `go get helm.sh/helm/v3@v3.2.0` to get dependencies
+Run `go build -o libhelm.dylib -buildmode=c-shared main.go` to build
```

Comment:
just fyi...it's in https://github.com/broadinstitute/helm-scala-sdk#helm-scala-sdk as well..also probably worth mentioning this is only applicable if you're using mac

Reply:
...and perhaps add that one could use ```go build -o libhelm.so -buildmode=c-shared main.go``` on linux.